mantle/cmd/kola/testiso.go

```diff
@@ -75,31 +75,52 @@ func init() {
 	root.AddCommand(cmdTestIso)
 }
 
-type kernelSetup struct {
-	kernel, initramfs string
-}
-
 func runTestIso(cmd *cobra.Command, args []string) error {
 	if kola.CosaBuild == nil {
 		return fmt.Errorf("Must provide --cosa-build")
 	}
 
+	baseInst := platform.Install{
+		CosaBuildDir: kola.Options.CosaBuild,
+		CosaBuild:    kola.CosaBuild,
+
+		Board:    kola.QEMUOptions.Board,
+		Firmware: kola.QEMUOptions.Firmware,
+	}
+
+	if instInsecure {
+		baseInst.Insecure = true
+	}
+
+	tmpd, err := ioutil.TempDir("", "kola-testiso")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpd)
+
+	completionfile := filepath.Join(tmpd, "completion.txt")
```

Comment:
Is it possible for both the legacy & live tests to both run? If so would there be any problems with both using the same completion file?